modules/compiler/builtins/CMakeLists.txt

```diff
@@ -35,9 +35,14 @@ include(cmake/FindBuiltinsTools.cmake)
 if(NOT CA_EXTERNAL_BUILTINS AND CA_RUNTIME_COMPILER_ENABLED)
   if(NOT DEFINED CA_BUILTINS_TOOLS_DIR)
     if(CMAKE_CROSSCOMPILING)
-      message(FATAL_ERROR
-        "Cross compiled builds requires CA_EXTERNAL_BUILTINS or "
-        "CA_BUILTINS_TOOLS_DIR to be set!")
+      if(EXISTS ${LLVM_NATIVE_TOOL_DIR})
+        set(CA_BUILTINS_TOOLS_DIR ${LLVM_NATIVE_TOOL_DIR})
+      else()
+        message(FATAL_ERROR
+          "Cross compiled builds requires CA_EXTERNAL_BUILTINS or "
+          "CA_BUILTINS_TOOLS_DIR to be set, or set LLVM_NATIVE_TOOL_DIR"
+          "if you are building in-tree.")
```

Comment:
`LLVM_NATIVE_TOOL_DIR` is a DPC++-specific variable and I don't think we fully support in-tree builds done by users just yet, so I think it's fine to use the variable if it is set, but we probably should not include it in the message we tell users (and it might be good to include a CMake comment that this is specifically for DPC++)

Reply:
Yeah you are right, I've removed the mention of `LLVM_NATIVE_TOOL_DIR`, thank you